src/psyclone/tests/test_files/dynamo0p3/infrastructure/lfric_include_flags.inc

```diff
@@ -0,0 +1,4 @@
+DIRS := $(subst $(LFRIC_PATH),,\
```

Comment:
Oops - indeed. Sorry, I ran into a strange jenkins build issue: my original solution was to have a special Makefile target which other makefiles would query (`INFLUDE_FLAGS :- $(shell make ...)`. This worked fine locally after I added the '-s' flag to  avoid that makefile verbose output ('Entering directory...'). This solution is much better than the include file, since then the infrastructure makefile does not rely on some variables suddenly defined by an include files (which you correctly pointed out was missing in documentation).

But for reasons I do not understand in jenkins all this verbose output still made it into the output (resulting in the compiler trying to make sense of the parameters "Entering directory..."  😢 ). After a while I gave up and used an include file :(

Reply:
Thanks.